================================================================================================================================================================================================================================
============== WRONG TESTCASE: _P16___ ==========================================================================================================================================================
================================================================================================================================================================================================================================


------------------------------------------------
|            Testcase's content                |
------------------------------------------------
main: function integer()
{
    for (1 = 1, i < 10, i + 1) foo();
}
------------------------------------------------
|              Expected output                 |
------------------------------------------------
something else
------------------------------------------------
|                Your output                   |
------------------------------------------------
Error on line 4 col 1: <EOF>

(program 

    (declaration 

        (function_declaration main : function 

            (type_specifier integer) 

            ( ) 

            (function_body 

                (block_statement { 

                    (statement 

                        (for_statement for 

                            ( 1 = 1 ,)) statement 

                        (statement i < 10 ,) statement 

                        (statement i + 1 )) 

                    (statement 

                        (call_statement foo 

                            ( ) ;)) })))) <EOF>)